stable/pipelines/templates/argo/configmap.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.rbac.create }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: workflow-controller-configmap
```

Comment:
All names should be derived from release name (by convention and also allowed us to know what's what and control the resource names more easily)

So this can be for example:
`name: {{ .Release.Name }}-workflow-controller-configmap`


Leaving it for you to decide whether a configmap needs `configmap` in it's resource name. I think it's redundant, some like it

Reply:
done